scripts/build/mirror.test.ts

```diff
@@ -176,6 +176,146 @@ describe('mirror', () => {
       });
     });
 
+    describe('Ranges', () => {
+      it('chrome_android 18', () => {
+        const support: InternalSupportBlock = {
+          chrome: {
+            version_added: '≤10',
+          },
+        };
+
+        const mirrored = mirrorSupport('chrome_android', support);
+        // No range because this is the first release.
+        assert.deepEqual(mirrored, { version_added: '18' });
+      });
+
+      it('chrome_android', () => {
+        const support: InternalSupportBlock = {
+          chrome: {
+            version_added: '≤100',
+          },
+        };
+
+        const mirrored = mirrorSupport('chrome_android', support);
+        assert.deepEqual(mirrored, { version_added: '≤100' });
+      });
+
+      it('edge 79', () => {
+        const support: InternalSupportBlock = {
+          chrome: {
+            version_added: '≤10',
+          },
+        };
+
+        const mirrored = mirrorSupport('edge', support);
+        // Uncertain early Chrome support doesn't imply possible EdgeHTML
```

Comment:
I was a bit concerned about introducing ranges where there might not be any uncertainty, so I looked at what would be different if Edge 79 doesn't get ranges. Happily, only these 22 paths are affected:

css.properties.filter.svg_elements
html.elements.iframe.sandbox.allow-forms
html.elements.iframe.sandbox.allow-pointer-lock
html.elements.iframe.sandbox.allow-same-origin
html.elements.iframe.sandbox.allow-scripts
html.elements.iframe.sandbox.allow-top-navigation
http.data-url.html_files
http.headers.Content-Security-Policy.form-action.blocks_redirects
http.mixed-content
http.mixed-content.allow_file_urls
http.mixed-content.allow_localhost_url
http.mixed-content.allow_loopback_url
svg.global_attributes.fill-rule
svg.global_attributes.filter
webextensions.api.permissions.Permissions
webextensions.api.permissions
webextensions.api.permissions.contains
webextensions.api.permissions.getAll
webextensions.api.permissions.onAdded
webextensions.api.permissions.onRemoved
webextensions.api.permissions.remove
webextensions.api.permissions.request

Fixing the data in these cases is probably better than adding a rule about transitions from non-WebKit to Blink engines.

Reply:
Some ranges removed in https://github.com/mdn/browser-compat-data/pull/23858.

Reply:
Another in https://github.com/mdn/browser-compat-data/pull/23861.

Reply:
https://github.com/mdn/browser-compat-data/pull/23865 fixes `filter` for SVG elements. That leaves only the HTTP, SVG and WebExtensions data.